examples/plot_benchmark_DL.py

```diff
@@ -0,0 +1,98 @@
+"""
+====================================================================
+Benchmarking with MOABB with DeepLearning architecture in Tensorflow
+====================================================================
+This example shows how to use MOABB to benchmark a set of DeepLearning pipeline (Tensorflow)
+on all available datasets.
+For this example, we will use only one dataset to keep the computation time low, but this benchmark is designed
+to easily scale to many datasets.
+"""
+# Authors: Igor Carrara <igor.carrara@inria.fr>
+#
+# License: BSD (3-clause)
+
+import os
+import random
+
+import matplotlib.pyplot as plt
+import numpy as np
+import tensorflow as tf
+from tensorflow import keras
+
+from moabb import benchmark, set_log_level
+from moabb.analysis.plotting import score_plot
+
+
+set_log_level("info")
+
+import absl.logging
```

Comment:
Can you move this import?
```suggestion

import matplotlib.pyplot as plt
import numpy as np
import tensorflow as tf
from tensorflow import keras
from absl.logging import set_verbosity, ERROR

from moabb import benchmark, set_log_level
from moabb.analysis.plotting import score_plot

set_log_level("info")
```